app/models/planning_application.rb

```diff
@@ -444,7 +446,7 @@ def set_reference
   end
 
   def set_key_dates
-    self.expiry_date = 56.days.after(validated_at || received_at)
+    self.expiry_date = DAYS_TO_EXPIRE.days.after(validated_at || received_at)
     self.target_date = 35.days.after(validated_at || received_at)
```

Comment:
Hey any reason why we wouldn't update this to a constant too then ?

Reply:
The 35? It should be changed.

But I didn't know enough about it to give it a name and then I would have to use it in the tests. My idea would be to do the change separately later when I would change it and change the tests it is involved with.

Reply:
Yeah exactly, I agree with moving it to a constant and making it consistent so change separately sounds good 🙌 .

Perhaps just `DAYS_TO_TARGET`  ? But you're right I'm not even sure we use this date really or it doesn't have much significance anyway in the business context.

I think at some point we were going to do an audit of the code to pull out these dates and create some documentation like we've done with notifications so we have a better oversight and can manage it better going forward